packages/terra-form-checkbox/docs/README.md

```diff
@@ -0,0 +1,26 @@
+# Terra Form Checkbox
+
+The Terra Form Checkbox is a responsive input component rendered as a box. When activated, a check mark shall appear. Focus can be activated through tabbing and the checked state can be toggled with the space bar.
```

Comment:
nit: rendered as a box next to the input value? 

Just rendered as a box seems a bit weird